src/sentry/models/group.py

```diff
@@ -346,12 +346,15 @@ def save(self, *args, **kwargs):
         )
         super().save(*args, **kwargs)
 
-    def get_absolute_url(self, params=None, event_id=None):
+    def get_absolute_url(self, params=None, event_id=None, organization_slug=None):
```

Comment:
If this is called on n groups you'd end up with getting 2n queries (1 per org and 1 per project)
![image](https://user-images.githubusercontent.com/4205004/112559583-a27ba880-8da7-11eb-8e8a-5bbdcf7d0079.png)

Could do `prefetch_related("project")` but this cuts it down to 1 query per project, but trace view queries across all projects so this could still be problematic
<img width="1159" alt="image" src="https://user-images.githubusercontent.com/4205004/112559611-b8896900-8da7-11eb-9fe1-457de2c2ec54.png">